lib/splash.js

```diff
@@ -3,6 +3,8 @@ var _ = require("lodash");
 var level = 0;
 var _logger;
 
+var KEYS_TO_HIDE = ["pass", "secret"];
```

Comment:
There is another key to hide _**password**_ (used in node-tech-db to configure the access to the database with the property _**db.connection.password**_)